web/src/routes/SurveyPages/SurveyListPage.tsx

```diff
@@ -1,5 +1,201 @@
+  }
+`;
+
+const Authlist = styled(Item)`
+  min-width: 160px;
+  width: 20vw;
+`;
+
+const EndDate = styled(Item)`
+  min-width: 150px;
+  width: 13vw;
+  text-align: center;
+  @media screen and (max-width: 900px) {
+    display: none;
+    }
+  } 
+`;
+
+const HeadTitle = styled(HeadItem)`
+  min-width: 15vh;
+  flex: 1;
+`;
+
+const HeadAuthList = styled(HeadItem)`
+  min-width: 160px;
+  width: 20vw;
+`;
+
+const HeadEndDate = styled(HeadItem)`
+  min-width: 150px;
+  width: 13vw;
+  @media screen and (max-width: 900px) {
+    display: none;
+    }
+  } 
+`;
+
+const Auth = styled.label`
+  font-size: 11px;
+  color: black;
+  padding: 5px;
+  border-radius: 7px;
+  margin: 3px;
+`;
+
+const Kakao = styled(Auth)`
+  background-color: #f7e600;
+`;
+const Google = styled(Auth)`
+  background-color: #ea4335;
+`;
+const Webmail = styled(Auth)`
+  background-color: #f3943e;
+`;
+const Id = styled(Auth)`
+  background-color: #edd8d3;
+`;
+const MobilePhone = styled(Auth)`
+  background-color: #a1daf7;
+`;
+const DriverLicense = styled(Auth)`
+  background-color: #34a853;
+`;
+const AuthNone = styled(Auth)`
+  background-color: #b3b2b2;
+`;
+
+const AuthSummary = styled.span`
+  font-size: 13px;
+`;
+interface Survey {
+  survey_id: string;
+  author: number;
+  title: string;
+  description: string;
+  created_date: string;
+  ended_date: string;
+  required_authentications: Array<string>;
+}
 
 export default function SurveyListPage() {
-  return <div>SurveyListPage</div>;
+  const [theme, toggleTheme] = useTheme();
+  const [surveys, setSurveys] = useState<Survey[]>([]);
+  const navigate = useNavigate();
+
+  const fetchSurveyData = async (): Promise<AxiosResponse<Survey[]>> => {
+    const request: AxiosResponse<Survey[]> = await axios.get<Survey[]>(requests.fetchSurveyListPage + 1);
+    setSurveys(request.data);
+    return request;
+  };
+
+  const makeAuthLabel = (auth: string) => {
+    switch (auth) {
+      case 'KAKAO':
+        return <Kakao key={auth}>카카오계정</Kakao>;
+      case 'GOOGLE':
+        return <Google key={auth}>구글계정</Google>;
+      case 'WEBMAIL':
+        return <Webmail key={auth}>학교인증</Webmail>;
+      case 'ID':
+        return <Id key={auth}>신분증</Id>;
+      case 'MOBILE_PHONE':
+        return <MobilePhone key={auth}>휴대폰인증</MobilePhone>;
+      case 'DRIVER_LICENSE':
+        return <DriverLicense key={auth}>운전면허</DriverLicense>;
+      default:
+        return <AuthNone key={auth}>Undefined</AuthNone>;
+    }
+  };
+  useEffect(() => {
+    fetchSurveyData();
+  }, []);
+
+  return (
+    <Container theme={theme}>
+      <Header theme={theme} toggleTheme={toggleTheme} />
+      <ListContainer theme={theme}>
+        <ListColumn>
+          <HeadTitle theme={theme}>설문 제목</HeadTitle>
+          <HeadAuthList theme={theme}>필수인증</HeadAuthList>
+          <HeadEndDate theme={theme}>설문 종료일</HeadEndDate>
+        </ListColumn>
+
+        {surveys.slice(0, 8).map((survey) => (
```

Comment:
Could you add a component like `No data` when there's no survey list?

Reply:
Ok! I'll add with pagination.